### ISSUE TITLE
Stories without empty titles should not be valid.

### ISSUE THREAD

**Issue by MattHeard** [MattHeard/Dendrite#12]
To reproduce:
1. Visit `index.jsp`.
2. Click **Start a new story**.
3. Make sure the **Title** field is empty.
4. Click **Submit**.
5. A page is successfully created with a blank level 2 header (`<h2></h2>`).

What should actually happen:

After step 4, the page should redirect back to the form with all of the elements restored to their submitted state and an error should pop up saying **Must not be blank** next to the **Title** field.
